### DISCUSSION TITLE
language problems

### DISCUSSION THREAD

**Discussion by MartinBjerg** [woocommerce/woocommerce#50004] [Checkout Flow]
I have challenges with "Add a coupon" not being automatically translated to my language (Danish) and it cannot be found in Loco translation.

**Comment by mikejolley** [comment]
Reading this document, Loco Translate does not extract strings from JavaScript by default: https://localise.biz/wordpress/plugin/manual/extracting#js

The string is translated however: https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/da/default/?filters%5Bterm%5D=Add+a+coupon&filters%5Bterm_scope%5D=scope_any&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc

Mine wasn't translated at first, but going to Dashboard > Updates and updating translations there fixed it:

![Screenshot 2024-08-05 at 11 05 37](https://github.com/user-attachments/assets/f018b788-ead4-493d-97cf-db2e55444f69)

**Comment by MartinBjerg** [comment]
Hello Mike

Thank you for your response. I have found a way to modify the source code
with the help of AI (GPT), but in the meantime, WooCommerce has released an
update that allows for the addition of translations for the specific areas.

Best regards, Martin.

Den man. 5. aug. 2024 kl. 12.06 skrev Mike Jolley ***@***.***
